pkgs/top-level/beam-packages.nix

```diff
@@ -12,24 +13,28 @@
 let
   self = beam;
 
+  versionAvailables = [ "25" "26" "24" ];
+
   # Aliases added 2023-03-21
-  versionLoop = f: lib.lists.foldr (version: acc: (f version) // acc) { } [ "26" "25" "24" ];
+  versionLoop = f: lib.lists.foldr (version: acc: (f version) // acc) { } versionAvailables;
 
   interpretersAliases = versionLoop (version: {
-    "erlangR${version}" = self.interpreters."erlang_${version}";
-    "erlangR${version}_odbc" = self.interpreters."erlang_${version}_odbc";
-    "erlangR${version}_javac" = self.interpreters."erlang_${version}_javac";
-    "erlangR${version}_odbc_javac" = self.interpreters."erlang_${version}_odbc_javac";
+    "erlangR${version}" = self.interpreters."erlang_${version}" or pkgs."erlang_${version}";
+    "erlangR${version}_odbc" = self.interpreters."erlang_${version}_odbc" or (throw "${pkgs."erlang_${version}"} is available ?");
+    "erlangR${version}_javac" = self.interpreters."erlang_${version}_javac" or (throw "${pkgs."erlang_${version}"} is available ?");
+    "erlangR${version}_odbc_javac" = self.interpreters."erlang_${version}_odbc_javac" or (throw "${pkgs."erlang_${version}"} is available ?");
```

Comment:
I hesitate between this way of writing (alias stay in aliases.nix) or use [inline aliases](https://github.com/NixOS/nixpkgs/blob/21d3b6284c75a138ecf35b948eaa58ca8ab879e3/pkgs/desktops/pantheon/default.nix#L249)